app/src/main/java/com/example/kotlin_java_practicalss/AbstractClass_demo.java

```diff
@@ -0,0 +1,33 @@
+package com.example.kotlin_java_practicalss;
+
+abstract class jaguar
```

Comment:
Naming of Class should have CamelCase. also the starting brace written at same line of class declaration,
abstract class Jaguar {